crates/wasmtime/src/module.rs

```diff
@@ -953,6 +948,14 @@ impl Module {
     pub fn engine(&self) -> &Engine {
         &self.inner.engine
     }
+
+    /// Returns the `ModuleInner` cast as `ModuleRuntimeInfo` for use
+    /// by the runtime.
+    pub(crate) fn runtime_info(&self) -> Arc<dyn wasmtime_runtime::ModuleRuntimeInfo + 'static> {
+        // N.B.: this needs to return a clone because we cannot
+        // statically cast the Arc<ModuleInner> to Arc<dyn Trait...>.
```

Comment:
I think this comment may be missing leading `&` on the types because otherwise this is saying that the cast happening here is the one that's not possible.